python/example_code/ses/ses_replicateidentities/ses_replicateidentities.py

```diff
@@ -0,0 +1,343 @@
+                                     {"Name": dkim_tok+"._domainkey."+dkim_dom,
+                                      "Type": "CNAME", "TTL": 1800,
+                                      "ResourceRecords":
+                                      [{"Value": dkim_tok+".dkim.amazonses.com"}
+                                       ]
+                                      }
+                                     }
+                                    ]
+                         }
+                add_txt = r53.change_resource_record_sets(
+                                                          HostedZoneId=zone_id,
+                                                          ChangeBatch=batch
+                                                         )
+                print(add_txt)
+            else:
+                pass
+
+
+# Enable DKIM and add CNAME records for verification.
+def generate_dkim(identity, region):
+    ses_dest_client = S.client('ses', region_name=region)
+    ask = input("Do you want to configure DKIM for "+identity+"? (yes/no)")
+    if ask == 'yes':
+        if '@' in identity:
+            dkim_tokens = ses_dest_client.\
+                         verify_domain_dkim(
+                                            Domain=identity.\
+                                                   split('@')[1])['DkimTokens']
+            if r53dom == 'no':
+                print("Add the following DKIM tokens as \
+                      CNAME records through your DNS provider:")
+                print(dkim_tokens)
+            elif r53dom == 'yes':
+                for token in dkim_tokens:
+                    # Then add CNAME records
+                    add_route_53_record(rec_type='dkimVerify',
+                                        dkim_dom=identity.split('@')[1],
+                                        dkim_tok=token
+                                        )
+        else:
+            dkim_tokens = ses_dest_client.\
+                          verify_domain_dkim(
+                                             Domain=identity)['DkimTokens']
+            if r53dom == 'no':
+                print("Add the following DKIM tokens as \
+                      CNAME records through your DNS provider:")
+                print(dkim_tokens)
+            elif r53dom == 'yes':
+                for token in dkim_tokens:
+                    # Then add CNAME records
+                    add_route_53_record(rec_type='dkimVerify',
+                                        dkim_dom=identity,
+                                        dkim_tok=token
+                                        )
+    elif ask == 'no':
+        return
+
+
+# Add SNS topic for bounces, deliveries, and complaints for a single identity.
+def sns_topics(identity, region):
+    ses_dest_client = S.client('ses', region_name=region)
+    ask = input("Do you want to configure an ",
+                "SNS topic for "+identity+"? (yes/no)")
```

Comment:
Syntax error (extra comma in line 226) causes this code to fail.

Reply:
Have taken out the comma and changed this to a single line of code.